environment/stats/src/cloud/benchflow/collectors/stats.go

```diff
@@ -154,6 +154,21 @@ func stopCollecting(w http.ResponseWriter, r *http.Request) {
 	close(stopChannel)
 	waitGroup.Wait()
 	collecting = false
+	composedMinioKey := ""
+	for _, container := range containers {
+		//minio.GzipFile("./"+container.ID+"_tmp")
+		minio.GzipFile("/app/"+container.ID+"_tmp")
```

Comment:
Where are you generating this file? On line [85](https://github.com/benchflow/collectors/pull/47/files#r54956202) you now store the file in `./`